Library/Homebrew/formula.rb

```diff
@@ -405,11 +418,30 @@ def linked_keg
     Pathname.new("#{HOMEBREW_LIBRARY}/LinkedKegs/#{name}")
   end
 
+  def latest_head_prefix
+    head_prefixes = installed_prefixes.select do |p|
+      p.basename.to_s.start_with?("HEAD")
+    end
+
+    head_prefixes.sort_by do |p|
+      Tab.for_keg(Keg.new(p)).source_modified_time
+    end.last
```

Comment:
This piece seems to be changed in the other opened PR with `HEAD` versions. I just didn't changed this PRs commits. When https://github.com/Homebrew/brew/pull/478 is merged I'll just remove some of the commits and everything will be fine.